backend/internal/handlers/classrooms/assignments/works/works.go

```diff
@@ -35,32 +37,76 @@ func (s *WorkService) getWorksInAssignment() fiber.Handler {
 // Returns the details of a specific student work.
 func (s *WorkService) getWorkByID() fiber.Handler {
 	return func(c *fiber.Ctx) error {
-		work, err := getWork(s, c)
+		work, err := s.getWork(c)
 		if err != nil {
 			return err
 		}
+
+		feedback, err := s.store.GetFeedbackOnWork(c.Context(), work.ID)
+		if err != nil {
+			return errs.InternalServerError()
+		}
+
 		return c.Status(200).JSON(fiber.Map{
 			"student_work": work,
+			"feedback":     feedback,
 		})
 	}
 }
 
 func (s *WorkService) gradeWorkByID() fiber.Handler {
 	return func(c *fiber.Ctx) error {
-		work, err := getWork(s, c)
+		// get the work first
+		work, err := s.getWork(c)
 		if err != nil {
 			return err
 		}
 		if work.RepoName == nil || work.SubmittedPRNumber == nil {
 			return errs.BadRequest(errors.New("work has not been submitted for grading yet"))
 		}
 
-		var requestBody models.PRReview
+		// get TA user id
+		userClient, err := middleware.GetClient(c, s.store, s.userCfg)
+		if err != nil {
+			return errs.AuthenticationError()
+		}
+		TAGHUser, err := userClient.GetCurrentUser(c.Context())
```

Comment:
this is a questionable name. I don't think variables should be capitalized unless it's a special case